pkg/i2gw/emitters/common_emitter/emitter.go

```diff
@@ -14,27 +14,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package nginx
+package commonemitter
```

Comment:
nit: common_emitter ? is this an emitter we are going to use? if not, better to not call the package x_emitter. Maybe emitter_helpers or something idk

Reply:
I'll add some comments TODOs. I think `common` its better because it runs for everything. base suggests that every emitter inherits/derives from it.